### DISCUSSION TITLE
How to disable Next Page completely?

### DISCUSSION THREAD

**Discussion by Heus-Sueh** [vuejs/vitepress#3395] [Q&A]
My next page is not working well and I would like to disable it, how do I do that?

**Comment by sharpchen** [comment]
```ts
export default defineConfig({
  // ...
  themeConfig: {
        // ...
    docFooter: {
      prev: false,
      next: false,
    },
}
});
```

**Reply by Heus-Sueh** [reply]
It didn't work apparently, this is my code:
https://bin.disroot.org/?4e2920d1974b64f5#k2DBkqdodJdxX2Yk4qUWPzUDu3kHQoivezqPDL954Pb

or does this have to be defined in index.js?

**Reply by sharpchen** [reply]
It should be correct. At least for my rc25 project. https://vitepress.dev/reference/default-theme-config#docfooter

**Reply by Heus-Sueh** [reply]
strange, mine is still enabled for some reason

**Comment by Heus-Sueh** [comment]
It has to be outside the theme config
```
 docFooter: {
      prev: false,
      next: false,
  },
 themeConfig: {
```

**Reply by Heus-Sueh** [reply]
In fact, my version was outdated so it wasn't working

**Reply by Heus-Sueh** [reply]
the docfooter has to be inside the theme config
